src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -100,8 +100,11 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
-        List<Note> updatedNotes = editPersonDescriptor.getNotes().orElse(personToEdit.getNotes());
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags, updatedNotes);
+        List<Note> updatedStrengthNotes = editPersonDescriptor.getMiscellaneous().orElse(personToEdit.getStrengths());
+        List<Note> updatedWeaknessNotes = editPersonDescriptor.getMiscellaneous().orElse(personToEdit.getWeaknesses());
```

Comment:
Not sure if should be
```suggestion
        List<Note> updatedStrengthNotes = editPersonDescriptor.getStrength().orElse(personToEdit.getStrengths());
        List<Note> updatedWeaknessNotes = editPersonDescriptor.getWeakness().orElse(personToEdit.getWeaknesses());
```

Reply:
Thank you very much for correcting these and reminding me